js/home.js

```diff
@@ -17,7 +17,81 @@ inputField.addEventListener("keypress", (e) => {
     }
   }
 });
+////////////////////fetch the API of  the city and cuntry
+fetch("https://countriesnow.space/api/v0.1/countries/population/cities")
+    .then(
+        e => e.json()//json the data
+    )
```

Comment:
the `e` is the response from the API fetch ! change the return data  to a descriptive name instead of `e`